Sun_Shengran/progress.md

```diff
@@ -36,22 +36,22 @@ Week 5 Plan:
 
 Week 6 Plan:
 - [ ] Implement feature to allow users to delete selected items from recycle bin
-- [ ] Implement feature to allow users to delete all courses/sessions/students from recycle bin
+- [ ] Implement feature to allow users to delete all courses/sessions from recycle bin
 - [ ] Build UI presentation for deleting items
 
 Week 7 Plan:
-- [ ] Add tests for "Delete" & "Clear All" features
+- [ ] Add tests for "Delete Permanently" & "Delete All Courses/Sessions" features
 - [ ] Complete any work left from the last week
 
 Week 8 Plan:
-- [ ] Implement feature to allow users to restore selected courses (including corresponding sessions & students) from recycle bin
+- [ ] Implement feature to allow users to restore selected/all courses (including corresponding sessions & students) from recycle bin
 - [ ] Build UI presentation for restoring items
-- [ ] Add tests for "Restore" of courses features
+- [ ] Add tests for "Restore" & "Restore All Courses" of courses features
 
 Week 9 Plan:
-- [ ] Implement feature to allow users to restore selected sessions/students (if corresponding courses are not deleted) from recycle bin
+- [ ] Implement feature to allow users to restore selected/all sessions and student responses (if corresponding courses are not deleted) from recycle bin
```

Comment:
In fact, will students who gave these student responses be restored as well?

Reply:
Same as the above, students won't be deleted if only a session has been deleted instead of the corresponding course.